cmd/mavenBuild.go

```diff
@@ -61,7 +109,13 @@ func runMavenBuild(config *mavenBuildOptions, telemetryData *telemetry.CustomDat
 	}
 
 	if config.CreateBOM {
-		goals = append(goals, "org.cyclonedx:cyclonedx-maven-plugin:2.7.8:makeBom", "org.cyclonedx:cyclonedx-maven-plugin:2.7.8:makeAggregateBom")
+		// Separate run for makeBOM goal
+		if err := runMakeBOMGoal(config, utils); err != nil {
+			return errors.Wrap(err, "failed to execute makeBOM goal")
```

Comment:
Ideally, I think we should be migrating to fmt.Errorf as it's standard lib and errors package is not maintained.